tests/tensor/rewriting/test_math.py

```diff
@@ -2132,7 +2140,7 @@ def test_local_div_switch_sink(self):
                 f = self.function_remove_nan(
                     [condition[0], x[0], c], [y], mode=self.mode
                 )
-                if type(condition[1]) is list:
+                if isinstance(condition[1], list):
```

Comment:
Same

Reply:
All of them were made manually by me as ruff raised an error and suggested this solution 😬.